### ISSUE TITLE
[Feature Request] Display aliases for virtual nodes

### ISSUE THREAD

**Issue by Salamander230** [zsviczian/excalibrain#66]
### Issue
ExcaliBrain can currently display nodes for resolved links using YAML-specified aliases. However, if a link is unresolved, ExcaliBrain does not specify an alias for it since that link does not have associated YAML in a markdown file.

### Proposed Solution
Expand alias functionality in ExcaliBrain to recognize aliases written explicitly in the forms `[[fileName|alias]]` or `[alias](fileName.md)` when a link is unresolved. Otherwise, if a link is resolved, use the YAML-specified alias.

Thanks for your work on this awesome plugin!

**Comment by zsviczian** [comment]
Makes sense. What if an unresolved link appears in multiple documents but with a different alias in each?
